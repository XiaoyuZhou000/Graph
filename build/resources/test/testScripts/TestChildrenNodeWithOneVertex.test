# Graph Testing File.
# Tests the children node for a graph that only has one Vertex

# Create a graph
CreateGraph graph1

# Add a vertex
AddNode graph1 n1

# Print the outgoing edges from n1
ListChildren graph1 n1
