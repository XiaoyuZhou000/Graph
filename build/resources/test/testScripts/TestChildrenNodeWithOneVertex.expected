# Graph Testing File.
# Tests the children node for a graph that only has one Vertex

# Create a graph
created graph graph1

# Add a vertex
added node n1 to graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are:

